inventorys/views.py

```diff
@@ -1,3 +1,53 @@
-from django.shortcuts import render
+import json
+import re
+import bcrypt
+import jwt
 
-# Create your views here.
+from django.views     import View
+from django.http      import JsonResponse
+from django.db.models import Q
+
+from chickenfood.settings import SECRET_KEY
+from members.models       import Member
+from products.models      import Product, Option
+from inventorys.models    import Inventory
+from members.utils        import login_decorator
+
+class InventorysView(View):
+    @login_decorator
+    def post(self, request):
+        data = json.loads(request.body)
+        member = request.member.id
+ 
+        if not (data.get('product_id') and data.get('quantity')):
+            return JsonResponse({"message": "KEY_ERROR"}, status=400)
+
+        if not Product.objects.filter(id=data['product_id']).exists():
+            return JsonResponse({"message": "INVALID_VALUE"}, status=400)
+
+        product = Product.objects.get(id=data['product_id']).id
+
+        if Option.objects.filter(id=data.get('option_id')).exists():
+            option = Option.objects.get(id=data['option_id']).id
+        else:
+            option = None
```

Comment:
```suggestion
        option_id = data.get('option')
         
        if not Option.objects.filter(id=option_id).exists():
            option_id = None
```
이러면 조금 더 깔끔해 지지 않을까요?? 어떻게 생각하시요?